### PR TITLE
main layout added

### PR DESCRIPTION
**Pull request by jstsanam**
Problem statement: Main layout UI with all the sections like Artists, Albums, Charts, etc.

Preview:

https://github.com/user-attachments/assets/2fe27927-8de3-418a-a45a-68dc6d3927ca

### CODE REVIEW

File: src/components/main-panel/MainPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import MainPanelFooter from "./MainPanelFooter";
+
+export default function MainPanel() {
+  return (
+    <div
+      className="rounded-md mr-2 pl-4 pt-3 overflow-y-scroll"
+      style={{
+        background: "linear-gradient(to bottom, #212121 0%, #121212 100%)",
+        width: "79%",
+      }}
```
Comment (som-matrix) [inline src/components/main-panel/MainPanel.tsx:11]: use tailwindcss to handle this

File: src/components/main-panel/cardsData.ts [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+        subtitle: "Perfect music for perfect food.",
+      },
+      {
+        image: "/images/playlist/dinner-friends.jpg",
+        title: "",
+        subtitle: "The perfect soundtrack to those long nights over dinner",
+      },
+    ],
+  },
+];
```
Comment (som-matrix) [inline src/components/main-panel/cardsData.ts:213]: why this file is in components directory ? 
if this is static move this to a directory called `data`

File: src/components/main-panel/footerData.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    heading: "Spotify Plans",
+    data: [
+      "Premium Individual",
+      "Premium Duo",
+      "Premium Family",
+      "Premium Student",
+      "Spotify Free",
+    ],
+  },
+];
```
Comment (som-matrix) [inline src/components/main-panel/footerData.ts:24]: should be moved to the same directory called `data`

File: src/components/main-panel/cardsData.ts (hunk 2 above)
Comment (jstsanam) [reply src/components/main-panel/cardsData.ts:213]: Sure

File: src/components/main-panel/footerData.ts (hunk 3 above)
Comment (jstsanam) [reply src/components/main-panel/footerData.ts:24]: Got it.

**Pull request merged**